docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: '3'
+
+services:
+  client:
+    build:
+      context: .
+      dockerfile: Dockerfile.client
+    image: elifesciences/peerscout_client:${IMAGE_TAG}
+    command: /bin/sh -c exit 0
+
+  peerscout:
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args: 
+        commit: ${IMAGE_TAG}
+    image: elifesciences/peerscout:${IMAGE_TAG}
+    command: /bin/sh -c exit 0
+    depends_on:
+      - client
+
+  peerscout-base-dev:
```

Comment:
if you only needed to run the application locally without running tests, would these be needed? If not they could be moved to a `docker-compose.ci.yml` file. But I suspect they are needed to work on the project, so they are probably shared between `dev` and `ci`.

Reply:
as a developer, going the docker route (which is not how I developed it so far), I would want to run tests which will enabled in `peerscout-dev`. `peerscout` without suffix would be the server more or less run in production, which I might also want to test locally. Although this PR doesn't yet include running the server (could be a separate PR), which depends on config, database and data/cache directory.

Reply:
So the `dev` image/services should definitely stay for development, there may be a separation between `dev` and `ci` later on.